k8sfv/run-test

```diff
@@ -211,13 +211,27 @@ if test -n "${GINKGO_FOCUS}"; then
     docker exec ${prefix}-felix /bin/sh -c \
 	   "${run} -ginkgo.focus \"${GINKGO_FOCUS}\""
 elif ${JUST_A_MINUTE}; then
+    tmp=$(mktemp)
     # Run all the tests that are _not_ marked as "[slow]", and fail
     # the test if any of those takes longer than 2 minutes.  A k8sfv
     # test should be marked as "[slow]" if it normally takes longer
     # than 1 minute - so this allows a buffer of 1 extra minute to
     # allow for executor slowness.
     docker exec ${prefix}-felix /bin/sh -c \
-	   "${run} -ginkgo.skip \"\\[slow\\]\" -ginkgo.slowSpecThreshold 120" | perl -pe 'END { exit $status } $status=1 if /SLOW TEST|FAIL!/;'
+       "${run} -ginkgo.skip \"\\[slow\\]\" -ginkgo.slowSpecThreshold 120" | tee $tmp
+	# Fail if the docker command returned failure
+	test ${PIPESTATUS[0]} -eq 0
+	# Grab the lines that have SLOW TEST until a line with ---------
```

Comment:
You've got mixed tabs/spaces in this file, please can you fix up?  I usually go for 3-space indents in shell.

Reply:
It looked like the 'tabs' were 4 spaces in there so I standardized on that. If you want me to update it all to 3, I can do that if you think I should.

Reply:
Also, I typoed above, meant to say 2-space.  2 or 4 is fine.  3 would be crazy!